src/file/file.rpc.ts

```diff
@@ -1,10 +1,17 @@
-import { FileService } from './file.service';
-import { IFile } from './file.interface';
 import { ObjectID } from 'mongodb';
 import { GrpcHealthCheck, HealthCheckResponse, HealthService } from 'grpc-ts-health-check';
+import * as grpc from 'grpc';
+import * as protoLoader from '@grpc/proto-loader';
+import apm = require('elastic-apm-node');
```

Comment:
Use `import apm from 'elastic-apm-node';`.

Reply:
doesn't work for some reason. still gives me the typing.

Reply:
done.